src/queries/queries.js

```diff
@@ -170,14 +170,37 @@ mutation createFollower($userId: ID!, $friendId: ID!){
     userId: $userId
     friendId: $friendId
   }) {
-    follower {
+    followerInfo {
       id
-      createdAt
+      userName
```

Comment:
Nice structuring of these queries! They're quite long and get back a lot of info! Are we using all of it?